jx_sqlite/expressions.py

```diff
@@ -29,7 +29,7 @@
 
 @extend(Variable)
 def to_sql(self, schema, not_null=False, boolean=False):
-    cols = [c for cname, cs in schema.items() if startswith_field(cname, self.var) for c in cs]
+    cols = [c for cname, cs in schema.items() if startswith_field(get_property_name(cname), get_property_name(self.var)) for c in cs]
```

Comment:
Add a method to the schema that will return a map from the relative namespace (namespace from the "origin" as per https://github.com/mozilla/jx-sqlite/blob/master/docs/Perspective.md ) to the column.  In addition to the relative namespace, include the absolute namespace (`names['.']`) so either perspective is allowed.  Much like https://github.com/klahnakoski/ActiveData/blob/typed/jx_python/__init__.py#L161